### DISCUSSION TITLE
stuttering with Nvidia

### DISCUSSION THREAD

**Discussion by Goddard** [hertg/egpu-switcher#86] [Q&A]
Not sure if this is expected behavior, or it is just my laptop specifically, but when I use an eGPU I get stuttering on my desktop.

Not sure if it is a power delivery issue or something else.  My BIOs give be basically no options so I can't shut off my dedicated GPU or the iGPU.

It is an Aero 15 YD and it has a 3080 max-q dedicated GPU.  My eGPU is a razer chroma x.  It also has a 3080.

I've tried various things and it appears no matter what results seem to be the same.  Just curious if anyone has any recommendation for extra things I could try.

**Comment by T-X** [comment]
Just a long shot, maybe try a newer Linux kernel or Mesa version, if available? (which versions are you using right now?)

For the AMD/ATI side I got an interesting link to a very recent patch for an eGPU optimization/fix: https://cgit.freedesktop.org/mesa/mesa/commit/?id=95d06343c693aa12b4cda5cda31d81fae138b0ec. Which is part of the the Mesa v23 release. Maybe there might be similar fixes/updates for Nvidia cards?

Does it make a difference to disable the laptop screen and to just use a display connected to the Razer Core X Chroma? Or the other way around, just using the laptop screen (while still having the eGPU performing the rendering jobs?).
